flight/drone/drone_controller.py

```diff
@@ -32,18 +34,23 @@ class DroneController(object):
         A PriorityQueue holding tasks to be performed.
     _safety_event : Event
         Set when an unsafe condition is observed.
+    _is_simulation : bool
+        Set to true when the code is intended for the simulator.
     """
 
-    def __init__(self, drone):
+    def __init__(self, drone, is_simulation=False):
         """Construct a drone controller.
 
         Parameters
         ----------
-        drone : c.Drone.{DRONE_NAME}"""
+        drone : c.Drone.{DRONE_NAME}
+        """
         self._task_queue = PriorityQueue()
         self._current_task = None
         self._safety_event = Event()
 
+        self._is_simulation = is_simulation
+
         # Initialize the logger
         self._logger = logging.getLogger(__name__)
         coloredlogs.install(level=logging.INFO)
```

Comment:
Logging-level should either be a file constant or be configurable through an argument passed to this function (preferably the latter)

Reply:
Added